hw/ip/rv_core_ibex_peri/data/rv_core_ibex_peri.hjson

```diff
@@ -45,13 +45,15 @@
     },
 
     { struct:  "region_cfg",
+      width:   2,
```

Comment:
Looks like we've fixed the same thing here ;) https://github.com/lowRISC/opentitan/pull/7230

However, I believe this needs an adjustment in the ibex intersignal declaration as well (in `top_earlgrey.hjson`) to work correctly. Otherwise topgen creates a partial one-to-N connection